verl/workers/fsdp_workers.py

```diff
@@ -833,7 +841,7 @@ def compute_ref_log_prob(self, data: DataProto):
         return output
 
     @register(dispatch_mode=Dispatch.ONE_TO_ALL)
-    def save_checkpoint(self, local_path, hdfs_path=None, global_step=0, max_ckpt_to_keep=None):
+    def save_checkpoint(self, local_path, remote_path=None, global_step=0, max_ckpt_to_keep=None):
```

Comment:
instead of making breaking changes, can we do this:
1. add an extra argument remote_path=None. keep hdfs_path as well. 
2. in the doc state that hdfs_path is deprecated.
3. in current codebase, always pass argument with remote_path
4. assert that remote_path and hdfs_path should not both be set

Reply:
add leave a record in https://github.com/volcengine/verl/discussions/2270 so that we can drop hdfs_path in the next release